install-deps.sh

```diff
@@ -397,6 +400,9 @@ else
                 elif test $ID = rhel -a $MAJOR_VERSION = 8 ; then
                     $SUDO subscription-manager repos --enable "codeready-builder-for-rhel-8-*-rpms"
                 fi
+                if test $MAJOR_VERSION = 7 ; then
+                    python_interpreters+=" python2.7"
+                fi
```

Comment:
I'm in favor of adding this code to the version(s) of Ceph that support CentOS 7. Does that include master/Octopus?

Reply:
It is _planned_ that master and Octopus will drop support for CentOS7

Reply:
before the test nodes are CentOS7-free, i don't dare to drop CentOS7 support. same applies to Ubuntu xenial.

Reply:
There was a lot of resistance to the idea of dropping centos7 for octopus, so we should keep our options open.